css-two-column-layout/styles.css

```diff
@@ -73,7 +73,6 @@ button {
   margin-top: -2px;
 }
 
-.row-column {
-  display: flex;
+.column {
```

Comment:
`column` has a very specific meaning in layout classes, this class name does not really describe its purpose, and will create confusion when intermixed with the `column` classes for sizing. You should rename this to `justify-content-center`